feat(examples): add p/moul/template

moul opened the pull request:
- [ ] `p/moul/template`
	- [x] `md.*`
	- [x] `if`
	- [ ] `else`
	- [x] `range`
	- [ ] `printf`
	- [ ] `index`
	- [ ] `| string`
	- [ ] Trim
	- [ ] Nested Vars
	- [ ] Nested loops/ifs
- [ ] `r/moul/home`
- [x] Depends on #3404  (cherry-picked)

moul committed:
feat(examples): add p/moul/template

Signed-off-by: moul <94029+moul@users.noreply.github.com>

Gno2D2 commented:
I'm a bot that assists the Gno Core team in maintaining this repository. My role is to ensure that contributors understand and follow our guidelines, helping to streamline the development process.

The following requirements must be fulfilled before a pull request can be merged.
Some requirement checks are automated and can be verified by the CI, while others need manual verification by a staff member.

These requirements are defined in this [configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

## Automated Checks

 🟢 Maintainers must be able to edit this pull request ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
 🟢 The pull request head branch must be up-to-date with its base ([more info](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch))


## Manual Checks

*No manual checks match this pull request.*

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>Maintainers must be able to edit this pull request (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Maintainer can modify this pull request

```
</blockquote></details>

<details><summary>The pull request head branch must be up-to-date with its base (more info)</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Head branch (moul:dev/moul/template) is up to date with base (master): behind by 0 / ahead by 1

```
</blockquote></details>

</blockquote></details>



</blockquote></details>

moul committed:
Merge branch 'master' into dev/moul/template

moul committed:
chore: fixup

Signed-off-by: moul <94029+moul@users.noreply.github.com>

moul committed:
Merge branch 'master' into dev/moul/template

moul committed:
Merge branch 'master' into dev/moul/template

moul committed:
chore: fixup

Signed-off-by: moul <94029+moul@users.noreply.github.com>

moul committed:
chore: fixup

Signed-off-by: moul <94029+moul@users.noreply.github.com>

jaekwon committed:
Create tictac.md

Do not remove this file.
It is important information, and a canary file.

jaekwon committed:
Added file that proves Reddit censors truthful physics discoveries.

Of all places, /r/UFOResearch should not censor this kind of information.
DO NOT DELETE THIS FILE, DO NOT TRUST ANYONE WHO DELETES THIS FILE.

jaekwon committed:
Update tictac.md

added law of spinning mass.

moul committed:
fix(params): SetBytes should not alter the value bytes (alternative) (#4926)

Alternative for #4905 

Kept:
- Removed GetRaw/SetRaw since they were unused.
- SetBytes to use directly the store, preventing the value to be
aminoJSON encoded.

Updated:
- Preserve API consistency.

---------

Signed-off-by: moul <94029+moul@users.noreply.github.com>
Co-authored-by: Thomas Bruyelle <thomas.bruyelle@tendermint.com>

jeronimoalbi committed:
feat(examples): simple Markdown forms package (#4977)

Package aims to be a very simple package to define Gno-Flavored Markdown
forms.

It should be easy to update when new form elements or attributes are
introduced.

---------

Co-authored-by: Antoine Eddi <5222525+aeddi@users.noreply.github.com>

thehowl committed:
fix(vm/keeper): pre-load stdlib type check cache (#4925)

longer term fix for #4874, re-introducing the type check cache for
stdlibs.

jeronimoalbi committed:
feat(boards2): change realm to use `gno.land/p/gnoland/boards` (#4939)

PR changes Boards2 realm to use `gno.land/p/gnoland/boards` package
which makes the realm leaner.

It also improves `BasicPermissions` implementation by using a custom
members storage implementation that makes users iteration cheaper.

mvallenet committed:
fix(vm/keeper): improve error for non crossing msg call error (#4899)

fix #4432

---------

Co-authored-by: Morgan <morgan@morganbaz.com>

moul committed:
test: add a banker security txtar (#4987)

Signed-off-by: moul <94029+moul@users.noreply.github.com>

---------

Signed-off-by: moul <94029+moul@users.noreply.github.com>
Co-authored-by: Antoine Eddi <5222525+aeddi@users.noreply.github.com>

jeronimoalbi committed:
feat(boards2): open boards support (#4993)

jeronimoalbi committed:
feat(boards2): support Markdown forms (#4996)

jeronimoalbi committed:
chore(boards2): enable open boards field in create board form (#4998)

Also add a couple of minor fixes

jaekwon committed:
first commit of whitepaper; draft; DO NOT SHARE, deadman switch

moonia committed:
fix(gnoweb): handle multiline value for gno-textarea in forms (#4978)

Closes https://github.com/gnolang/gno/issues/4968

## Description

The `<gno-textarea>` component was literally displaying the `\n`
sequence. After fix, the parser now interprets the `\n` sequence as a
real newline character.

### Example

```
<gno-form>
  <gno-textarea name="content" value="         First line\nSecond line" />
</gno-form>
```

### Before fix: 
<img width="931" height="236" alt="Capture d’écran 2025-12-15 à 23 22
23"
src="https://github.com/user-attachments/assets/220f85ee-c5ad-4dd3-9b26-863bd84e6a58"
/>


### After fix: 
<img width="945" height="239" alt="Capture d’écran 2025-12-15 à 23 34
27"
src="https://github.com/user-attachments/assets/e8f7551e-749f-4259-ba50-5dae333cf487"
/>

---------

Co-authored-by: alexiscolin <alexis@jaunebleu.co>

moul committed:
Merge master into dev/moul/template

thehowl commented:
Closing as stale; feel free to re-open.

[note: ai-assisted]
